ui/eureka/eureka.ts

```diff
@@ -43,42 +152,42 @@ const defaultOptions = {
       en: (nowMs, stopTime) => {
         if (stopTime) {
           const min = (stopTime - nowMs) / 1000 / 60;
-          return ' for ' + Math.ceil(min) + 'm';
+          return ` for ${Math.ceil(min)}m`;
         }
         return ' for ???';
       },
       de: (nowMs, stopTime) => {
         if (stopTime) {
           const min = (stopTime - nowMs) / 1000 / 60;
-          return ' für ' + Math.ceil(min) + 'min';
+          return ` für ${Math.ceil(min)}min`;
         }
         return ' für ???';
       },
       fr: (nowMs, stopTime) => {
         if (stopTime) {
           const min = (stopTime - nowMs) / 1000 / 60;
-          return ' pour ' + Math.ceil(min) + ' min ';
+          return ` pour ${Math.ceil(min)} min `;
         }
         return ' pour ???';
       },
       ja: (nowMs, stopTime) => {
         if (stopTime) {
           const min = (stopTime - nowMs) / 1000 / 60;
-          return ' 終わるまであと ' + Math.ceil(min) + ' 分 ';
+          return ` 終わるまであと ${Math.ceil(min)} 分 `;
```

Comment:
```suggestion
          return ` 終わるまであと${Math.ceil(min)}分 `;
```
It is a little weird to have spaces between numbers and `分`(minute), also there are no spaces needed in Japanese technically as they didn't affect the meaning, so just delete them here and everywhere.

Reply:
Yeah, I was just trying to replicate whatever was there before.  Thanks for taking a look.  It seemed weird to me too.